RequireThis/all-examples-in-one/list-of-projects.properties

```diff
@@ -37,3 +37,5 @@ protonpack|git|https://github.com/poetix/protonpack|protonpack-1.7||
 jOOL|git|https://github.com/jOOQ/jOOL|version-0.9.7||
 RxJava|git|https://github.com/ReactiveX/RxJava|v1.0.9||
 Vavr|git|https://github.com/vavr-io/vavr|v0.9.0||
+
+checkstyle|git|https://github.com/checkstyle/checkstyle.git|master|**/.ci-temp/**/*,**/resources-noncompilable/**/asttreestringprinter/**/*,**/resources-noncompilable/**/filefilters/**/*,**/resources-noncompilable/**/main/**/*,**/resources-noncompilable/**/suppressionsstringprinter/**/*,**/resources-noncompilable/**/gui/**/*,**/resources-noncompilable/**/javadocpropertiesgenerator/**/*,src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/javaparser/InputJavaParser.java,**/InputAllEscapedUnicodeCharacters.java,**/resources-noncompilable/**/javaparser/InputJavaParser.java,**/resources-noncompilable/**/checks/imports/unusedimports/InputUnusedImportsSingleWordPackage.java,**/resources-noncompilable/**/grammar/java19/*,**/resources-noncompilable/**/treewalker/**/*
```

Comment:
??
https://github.com/checkstyle/contribution/blob/d85c3ddc6ae531dc372b83509be88a1b15b86ac3/checkstyle-tester/projects-to-test-on-for-github-action.properties#L40-L41

Reply:
My bad has pushed again. Please let me know if any issue persists.